src/modules/dashboard/gameDev/gameDevHandlers/leaderboardGameDevHandlers.ts

```diff
@@ -30,6 +32,19 @@ export const getLeaderboardHandler: Lifecycle.Method = async (request, h) => {
   }
 };
 
+export const getLeaderboardResultsHandler: Lifecycle.Method = async (request, h) => {
+  const leaderboard = await getLeaderboardById(request.params.leaderboardId);
+
+  if (!leaderboard || !request.params.gameTypeId) {
+    throw Boom.notFound('Invalid leaderboard or gametypeId');
+  }
+
+  const leaderboardResult = await getLeaderboardResultRank(leaderboard);
```

Comment:
same thing here regrading the limit query string param. we should probably call the function like this:
await getLeaderboardResultRank(leaderboard, request.query.limit);